pages/task/list/router.js

```diff
@@ -27,13 +27,21 @@ module.exports = ({
     next();
   },
   locals: (req, res, next) => {
+    const progress = req.query.progress || tabs(req.user.profile)[0];
     const firstName = get(req, 'user.profile.firstName');
     const lastName = get(req, 'user.profile.lastName');
     res.locals.static.profileName = `${firstName} ${lastName}`;
     res.locals.static.progress = req.datatable.progress;
     res.locals.datatable.progress = req.datatable.progress;
 
     res.locals.static.tabs = tabs(req.user.profile);
+
+    if (progress === 'completed') {
+      res.locals.static.content.fields.updatedAt.label = 'Completed';
+    } else if (progress === 'inProgress' || (progress === 'outstanding' && !req.user.profile.asruUser)) {
```

Comment:
This is assuming a perfect flow through the system, but falls over if a task follows an unhappy path.

If a task is returned by a licensing officer to an inspector then it will appear in their `outstanding` tab, but the date will show when it was referred, not when it was submitted.

It feels to me like the label should always be either "Completed" or "Last changed" if the field that is being shown in the `updated_at`.

Reply:
Good point. I'll make it Last changed and you can explain to design ;)

Reply:
I'm fine with that.